model/cluster_message.go

```diff
@@ -26,6 +26,11 @@ const (
 	CLUSTER_EVENT_INVALIDATE_CACHE_FOR_SCHEMES                      = "inv_schemes"
 	CLUSTER_EVENT_CLEAR_SESSION_CACHE_FOR_ALL_USERS                 = "inv_all_user_sessions"
 
+	// Plugin Events
+	CLUSTER_EVENT_INSTALL_PLUGIN = "install_plugin"
+	CLUSTER_EVENT_REMOVE_PLUGIN  = "remove_plugin"
+
+	// ClusterMessage SendTypes
```

Comment:
nit: not a GoDoc comment -- remove?

Reply:
since these constants are not typed, a comment here to explain these are only applicable for `ClusterMessage.SendType` would help.